src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,8 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
 
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salary;
+
+        LocalDate from = LocalDate.parse(dateFrom, FORMATTER);
+        LocalDate to = LocalDate.parse(dateTo, FORMATTER);
+        StringBuilder info = new StringBuilder();
+        info.append("Report for period " + dateFrom + " - " + dateTo);
+
+        for (int i = 0; i < names.length; i++) {
+            salary = 0;
+            for (int k = 0; k < data.length; k++) {
+                String[] date = data[k].split(" "); // parse string to part "25.04.2019 John 60 50"
```

Comment:
the same with comments here) let's remove it)

Reply:
done!